js/code.js

```diff
@@ -359,3 +393,51 @@ function doDeleteUser(){
         }
     }
 }
+
+function doGoToAddContact()
+{
+        window.location.href = "createContacts.html";
+}
+
+function doUpdateContact()
```

Comment:
new doGoToAddContact() and doUpdateContact() function @cereusware @CathyChian @NicholasCG @winsonlin123